app/models/crop.rb

```diff
@@ -38,7 +38,11 @@ def search_data
   end
 
   def main_image_path
-    pictures.first.attachment.url if pictures.present?
+    if pictures.present?
+      pictures.first.attachment.url
+    else
+      ActionController::Base.helpers.asset_path('baren_field.jpg')
```

Comment:
Ah, never freaking mind. The fact that we are doing it via embedded document means we would need to add an extra build step to populate S3 and then add conditional logic to figure out which class its embedded in to return the correct URL.

If some reading this has a better idea, I'd love to hear it. In the meantime, I'm willing to look the other way and violate MVC in favor of having less code to maintain.